dashboard/app/models/pd/payment_terms.rb

```diff
@@ -0,0 +1,63 @@
+# == Schema Information
+#
+# Table name: pd_payment_terms
+#
+#  id                  :integer          not null, primary key
+#  regional_partner_id :integer
+#  start_date          :date
+#  end_date            :date
+#  course              :string(255)
+#  subject             :string(255)
+#  properties          :text(65535)
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#
+# Indexes
+#
+#  index_pd_payment_terms_on_regional_partner_id  (regional_partner_id)
+#
+
+class Pd::PaymentTerm < ApplicationRecord
+  include SerializedProperties
+
+  belongs_to :regional_partner
+
+  serialized_attrs %w(
+    per_attendee_payment
+    fixed_payment
+    minimum_attendees_for_payment
+    maximum_attendees_for_payment
+    pay_facilitators
+  )
+
+  def self.for_workshop(workshop)
+    return nil unless workshop.regional_partner
```

Comment:
So much for simplifying :(

Do you know if we have plans to combine these in some form? At least maybe we can remove most of the complexity in the old model and leave CSF, and district CSinA and CSinS